examples/blender/blender.py

```diff
@@ -61,11 +61,12 @@ async def worker(ctx: WorkContext, tasks):
             output_file = f"output_{frame}.png"
             ctx.download_file(f"/golem/output/out{frame:04d}.png", output_file)
             try:
-                # Set timeout for executing the script on the provider. Two minutes is plenty
-                # of time for computing a single frame, for other tasks it may be not enough.
-                # If the timeout is exceeded, this worker instance will be shut down and all
-                # remaining tasks, including the current one, will be computed by other providers.
-                yield ctx.commit(timeout=timedelta(seconds=120))
+                # Set timeout for executing the script on the provider. Usually, 30 seconds
+                # should be more than enough for computing a single frame, however a provider
+                # may require more time for the first task if it needs to download a VM image
+                # first. Once downloaded, the VM image will be cached and other tasks that use
+                # that image will be computed faster.
+                yield ctx.commit(timeout=timedelta(minutes=10))
```

Comment:
The requestor will have to wait 10 minutes if any provider is broken (there are currently providers that are signing agreements and then doing nothing).

Reply:
True. But the problem with the timeout being too low is recurring and sometimes affects all providers (for example it happened after `0.6.3` `yagna` release, when many providers needed to refresh their VM images and were unable to do so for a long time since the activities were interrupted by our `blender.py` requestor due to 2 minute timeouts).